Demo/Common/IntegrationExamples/Components/CardComponentExample.swift

```diff
@@ -0,0 +1,200 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import AdyenActions
+import AdyenCard
+import AdyenComponents
+import AdyenDropIn
+import PassKit
+import UIKit
+
+internal final class CardComponentExample: AdvancedFlowExampleProtocol {
+
+    internal var paymentMethods: PaymentMethods?
+    internal var cardComponent: PresentableComponent?
+    internal weak var presenter: PresenterExampleProtocol?
+
+    // MARK: - Action Handling
+
+    internal lazy var adyenActionComponent: AdyenActionComponent = {
+        let handler = AdyenActionComponent(context: context)
+        handler.configuration.threeDS.delegateAuthentication = ConfigurationConstants.delegatedAuthenticationConfigurations
+        handler.configuration.threeDS.requestorAppURL = URL(string: ConfigurationConstants.returnUrl)
```

Comment:
This part is optional. However, if this is "Advanced card Example" - we can keep it :)

Reply:
Okay.